pandas/io/excel/_base.py

```diff
@@ -1003,6 +1003,8 @@ class ExcelFile:
              then `odf <https://pypi.org/project/odfpy/>`_ will be used.
            - Otherwise if ``path_or_buffer`` is an xls format,
              ``xlrd`` will be used.
+           - Otherwise if ``path_or_buffer`` is in xlsb format,
```

Comment:
add a versionadded 1.3

Reply:
@jreback this is already part of a versionchanged block(1.2.0). Should i just update the versionchanged to 1.3.0?

Reply:
you can add a version updated 1.2 / alt i guess this is a backport so would be ok for 1.2.1